internal/osutils/autostart/autostart.go

```diff
@@ -49,10 +49,19 @@ func New(name AppName, exec string, args []string, options Options, cfg Configur
 	}, nil
 }
 
-func (a *app) Enable() error {
+func (a *App) Enable() error {
 	return a.enable()
 }
 
-func (a *app) Disable() error {
+func (a *App) Disable() error {
 	return a.disable()
 }
+
+// IsEnabled is provided for testing only.
+func (a *App) IsEnabled() (bool, error) {
```

Comment:
You're already violating the comment:

https://github.com/ActiveState/cli/pull/2403/files#diff-fe1d6fb736a8d64ee905da5161a57eae198ad600fe9e62e48390d0ad205dd88fR26

Why not just rename `isEnabled` to `IsEnabled`?

Reply:
This comment is for external callers only. It forwards to the isEnabled method because each OS has its own implementation. I've improved the linux isEnabled comment to emphasize this.